workers/unity/Assets/Playground/Scripts/Cubes/ScoreCollision.cs

```diff
@@ -0,0 +1,52 @@
+using Improbable.Gdk.Core;
+using UnityEngine;
+using Unity.Entities;
+using Generated.Playground;
+using Unity.Mathematics;
+
+[RemoveAtEndOfTick]
+public struct CollisionComponent : IComponentData
+{
+    public SpatialOSLaunchable other;
+}
+
+public class ScoreCollision : MonoBehaviour
+{
+    private MutableView view;
+    private SpatialOSLaunchable ownLaunchable;
+
+    public void Start()
+    {
+        var component = GetComponent<SpatialOSComponent>();
+        if (component)
+        {
+            view = WorkerRegistry.GetWorkerForWorld(component.World).View;
+            ownLaunchable = view.GetComponent<SpatialOSLaunchable>(component.Entity);
+        }
+        else
+        {
+            Debug.LogError("Could not retrieve SpatialOSComponent from game object.");
```

Comment:
We arent using Debug.LogError anymore, use the LoggingDispatcher.

Reply:
Oh I just realized that you can't do that given that you might not have access to a View. Ok ignore that comment.

Reply:
Will do!

Reply:
Actually, this check is not needed (it shouldn't ever fail), I think?

Reply:
It could if someone puts the `ScoreCollision` component on a prefab that is not instantiated by our systems - granted that's a user error, but the check is probably still worth having